modules/charcall.js

```diff
@@ -122,7 +122,11 @@ exports.hasData = function(client,charid,datatype){
    if(occList.length>0){
      for(i=0;i<occList.length;i++){
        if(occList[i][0]==false&&client.charcall.charData(client,occList[i][0],"control")=="NONE"){
```

Comment:
I'd probably fix this by just changing the occList check to a charData(faction) check, and only underlings, because we dont want prospitians/dersites blocking item pick-up either (at least for now).

Reply:
I'd decided to err on the side of caution, as I wasn't sure what factions existed, and which ones were or weren't intended to block card pickup. I think underlingCheck is also used to determine whether you're allowed to move in a dungeon.
But, looking over the code, it seems like Denizens belong to the underling faction, so... I think it's safe to make that change!

Reply:
yeah, for now only underlings will block movement and pick-up, and its one of those things that only makes sense if you've read the database a bunch. You got the check right, I was just shortening it for the sake of conversation. We can also make it even simpler, replacing the occList[i][0]==false check with the faction check, and I might even want to remove the control check, so that possessed underlings can still limit dungeon movement and item pick-up.